css-pseudo-4/Overview.bs

```diff
@@ -603,10 +603,32 @@ Default UA Styles</h3>
     /* Represent default spelling/grammar error styling in an adjustable way */
     :root::spelling-error { text-decoration-line: spelling-error; }
     :root::grammar-error  { text-decoration-line: grammar-error; }
-    /* Highlight targeted text */
-    :root::target-text    { color: MarkText; background: Mark; }
   </pre>
 
+  Note: the “paired cascade” behavior of [=highlight colors=]
+  means that UA defaults for these colors
+  can't be expressed in terms of 'color' and 'background-color'
+  in [=UA style sheet=] rules.
+  However it has been interoperably implemented in browsers
+  and is thus probably a Web-compatibility requirement.
+
+  UAs may define <dfn>default highlight colors</dfn>
+  for each highlight pseudo-element,
+  consisting of a 'color' and a 'background-color',
+  and doing so is recommended
+  for ''::selection'' and ''::target-text''.
```

Comment:
Is this an RFC2119 RECOMMEND? Or did you mean something different?

Reply:
4eb95b022c1eebb02bd4969bf723fd8ab4d6825a should make this relationship clearer.